super_gui/src/components/DoorManUI.tsx

```diff
@@ -43,12 +42,50 @@ const DoorManUI = () => {
     fetchBuildingLockers();
   }, []);
 
-  // Mock function to assign package
-  const assignPackage = () => {
-    if (!apartmentNumber || !selectedLocker) {
-      return;
+  // PATCH To Assign Packages
+  const assignPackage = async (lockerId: number, apartmentNumber: string) => {
+    try {
+      const res = await fetch(`http://localhost:8080/locker/${lockerId}`, {
```

Comment:
it is easier to change the base url for the backend if you start using the axios instances for your requests

Reply:
Now, we have to find all request in the code.